packages-ts/gauntlet-terra-contracts/src/commands/contracts/ocr2/proposal/acceptProposal.ts

```diff
@@ -30,7 +30,7 @@ const makeCommandInput = async (flags: any, args: string[]): Promise<CommandInpu
   const contract = args[0]
 
   return {
-    proposalId: flags.proposalId,
+    proposalId: flags.configProposal,
```

Comment:
Let's add support for both and add a comment that this is an ad-hoc alias as requested by eng ops.